TestCases/optimization_euler/equivalentarea_naca64206/NACA64206_direct.cfg

```
@@ -0,0 +1,160 @@
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+REF_LENGTH= 1
+REF_AREA= 0
+REF_DIMENSIONALIZATION= FREESTREAM_PRESS_EQ_ONE
+
+% ------------------------ SUPERSONIC SIMULATION ------------------------------%
+% MARKER_NEARFIELD needs to be defined on a circumferential boundary within
+% calculation domain so that it captures pressure disturbance from the model.
+% The boundary should have a structured grid with the same number of nodes
+% along each azimuthal angle.
+% To run inverse design using target equivalent area, TargetEA.dat is required.
+%
+% Evaluate equivalent area on the Near-Field (NO, YES)
+% 
+EQUIV_AREA= YES
+%
+% Integration limits of the equivalent area ( xmin, xmax, Dist_NearField )
+EA_INT_LIMIT= ( 234.854, 313.354, 220.38)
+%
+% Equivalent area scale factor ( EA should be ~ force based objective functions )
+EA_SCALE_FACTOR= 1.0
+%
+% Fix an azimuthal line due to misalignments of the near-field
+FIX_AZIMUTHAL_LINE= 90.0
+%
+% Drag weight in sonic boom Objective Function (from 0.0 to 1.0)
+DRAG_IN_SONICBOOM= 0.0
+
+% ----------------------- BOUNDARY CONDITION DEFINITION -----------------------%
+%
+MARKER_EULER= ( Wing )
+MARKER_FAR= ( Side )
+MARKER_SUPERSONIC_INLET=(Inlet,216.6499,11662.0,472.1153,0,0)
+MARKER_SUPERSONIC_OUTLET=(Outlet)
+MARKER_SYM= ( Symmetry )
+MARKER_PLOTTING= ( Wing )
+MARKER_MONITORING= ( Wing )
+%
+% Near-Field boundary marker(s) (NONE = no marker)
+MARKER_NEARFIELD= ( Nearfield )
+
+% ------------- COMMON PARAMETERS TO DEFINE THE NUMERICAL METHOD --------------%
+%
+NUM_METHOD_GRAD= GREEN_GAUSS
+
+% Objective function in gradient evaluation  (DRAG, LIFT, SIDEFORCE, MOMENT_X,
+%                                             MOMENT_Y, MOMENT_Z, EFFICIENCY,
+%                                             EQUIVALENT_AREA, NEARFIELD_PRESSURE,
+%                                             FORCE_X, FORCE_Y, FORCE_Z, THRUST,
+%                                             TORQUE, FREE_SURFACE, TOTAL_HEATFLUX,
+%                                             MAXIMUM_HEATFLUX, INVERSE_DESIGN_PRESSURE,
+%                                             INVERSE_DESIGN_HEATFLUX)
+OBJECTIVE_FUNCTION= EQUIVALENT_AREA
+%
+CFL_NUMBER=3
+CFL_ADAPT= NO
+%INNER_ITER= 4000
+INNER_ITER= 40
+LINEAR_SOLVER= FGMRES
+LINEAR_SOLVER_PREC=LU_SGS
+LINEAR_SOLVER_ERROR= 1E-10
+LINEAR_SOLVER_ITER= 50
+
+% ----------------------- SLOPE LIMITER DEFINITION ----------------------------%
+%
+SLOPE_LIMITER_FLOW=VAN_ALBADA_EDGE
+SLOPE_LIMITER_ADJFLOW=VAN_ALBADA_EDGE
+ADJ_SHARP_LIMITER_COEFF= 3.0
+REF_SHARP_EDGES= 3.0
+SENS_REMOVE_SHARP= YES
+
+% -------------------------- MULTIGRID PARAMETERS -----------------------------%
+%
+MGLEVEL= 0
+%
+% ----------------- ADJOINT-FLOW NUMERICAL METHOD DEFINITION ------------------%
+%
+CONV_NUM_METHOD_ADJFLOW= HLLC
+MUSCL_ADJFLOW=YES
+RELAXATION_FACTOR_ADJOINT= 1.0
+CFL_REDUCTION_ADJFLOW= 0.5
+TIME_DISCRE_ADJFLOW= EULER_IMPLICIT
+QUASI_NEWTON_NUM_SAMPLES= 20
+MG_ADJFLOW = NO
```

```suggestion
RELAXATION_FACTOR_ADJOINT= 1.0
QUASI_NEWTON_NUM_SAMPLES= 20
```
The options I removed are only used by the continuous adjoint. The discrete adjoint uses the normal numerical method definitions.